indico/modules/events/contributions/util.py

```diff
@@ -97,7 +98,8 @@ def generate_spreadsheet_from_contributions(contributions):
     """Return a tuple consisting of spreadsheet columns and respective
     contribution values"""
 
-    headers = ['Id', 'Title', 'Description', 'Date', 'Duration', 'Type', 'Session', 'Track', 'Presenters', 'Materials']
+    headers = ['Id', 'Title', 'Description', 'Date', 'Duration', 'Type', 'Session', 'Track', 'Presenters', 'Materials',
+               'Authors', 'Co-Authors', 'Boardnumber']
```

Comment:
Should we include the board number at all if there aren't any contributions with one?

Would be pretty easy to check for it..

```python
has_board_numbers = any(c.board_number for c in contributions)
```

Reply:
Sounds like a good idea. Done